cmd/monaco/main.go

```diff
@@ -148,8 +147,30 @@ func parseInputCommand(args []string, fileReader util.FileReader) (dryRun bool,
 
 	// Show usage if flags are invalid
 	if environmentsFile == "" {
-		println("Please provide environments yaml with -e/--environments!")
+		cliname := "Name: \n" +
+			"\tmonaco \n" +
+			"Version:\n" +
+			"\t" + version.MonitoringAsCode
+
+		cliuse :=
+			"\nUsage: \n" +
+				"\tmonaco -dry-run -environments <path-to-environment-yaml-file> -project <project-folder> [projects-root-folder] \n" +
```

Comment:
so far in our documentation we generally follow the unix style of double-dash `--` for the full flag, single-dash `-` for shorthand. 

While go flags allows both for everything, I think we should not promote it, and use `--` in these examples